docs/source/tutorial/type_check.rst

```diff
@@ -232,7 +231,7 @@ More complicated cases
 
 How to write a more complicated condition that can't be written with these operators?
 You can evaluate :class:`utils.type_check.Expr` and get its result value with :meth:`eval` method.
-And, check the condition and show warning message by your hand:
+Then check the condition and show warning message by hand:
```

Comment:
Then doesn't need a comma, unless there is a secondary clause after it. This would be like a second sentence giving more information; without it it is still a complete sentence. Like this:

"""
Then, since we know how to evaluate, check the condition and show warning message by hand.
"""